sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -664,7 +670,7 @@ public virtual AsyncPageable<ConfigurationSetting> GetConfigurationSettingsAsync
         }
 
         /// <summary>
-        /// Retrieves one or more <see cref="ConfigurationSetting"/> that satisfies the options of the <see cref="SettingSelector"/>
+        /// Retrieves one or more <see cref="ConfigurationSetting"/> that satisfy the options of the <see cref="SettingSelector"/>.
```

Comment:
ConfigurationSetting should be plural here, but it looks funny formatting-wise if we just put an 's' on the end.

What if we re-wrote it to something like "Retrieves one or more `ConfigurationSetting` entities that match the options specified in the passed-in `SettingSelector`" ?